robolaunch/robot-operator
# PR: Volume Provisioning

# :herb: PR: Volume Provisioning

## Description

Robot's main assets, volumes, are created over robot's definition.

## Type of change


## How can it be tested?

Deploy a robot with it's `.spec.storage` and `.spec.distro` are defined. After a little while, run `kubectl get pvc -n <NS>` to see dynamically provisioned volumes.
